adapters/src/main/kotlin/com/krzykrucz/elesson/currentlesson/infrastructure/Database.kt

```diff
@@ -0,0 +1,75 @@
+package com.krzykrucz.elesson.currentlesson.infrastructure
+
+import com.krzykrucz.elesson.currentlesson.domain.NaturalNumber
+import com.krzykrucz.elesson.currentlesson.domain.NonEmptyText
+import com.krzykrucz.elesson.currentlesson.domain.attendance.Attendance
+import com.krzykrucz.elesson.currentlesson.domain.attendance.AttendanceList
+import com.krzykrucz.elesson.currentlesson.domain.attendance.LessonTime
+import com.krzykrucz.elesson.currentlesson.domain.attendance.NotCompletedAttendance
+import com.krzykrucz.elesson.currentlesson.domain.startlesson.*
+import java.time.LocalDate
+import java.time.LocalTime
+import java.util.concurrent.ConcurrentHashMap
+
+class Database {
+
+    companion object {
+        private val lessonId1 = lessonIdOf("2019-09-09", 1, "1A")
+        private val lessonId2 = lessonIdOf("2019-09-09", 2, "1A")
+        private val lessonId3 = lessonIdOf("2019-09-09", 3, "1A")
+
+        private val classRegistryOf1A = ClassRegistry(
+                students = listOf(
+                        createStudentRecord("Harry", "Potter", 1),
+                        createStudentRecord("Tom", "Riddle", 2)
+
+                ),
+                className = classNameOf("1A")
+        )
+
+        val CLASS_REGISTRY_DATABASE: ConcurrentHashMap<ClassName, ClassRegistry> = ConcurrentHashMap(mutableMapOf(
+                classNameOf("1A") to classRegistryOf1A
+        ))
+
+
+        val SCHEDULE: ConcurrentHashMap<LessonHourNumber, LessonTime> = ConcurrentHashMap(mutableMapOf(
```

Comment:
this should be hardcoded as a part of the domain object LessonHourNumber, or even LessonHour

Reply:
done